components/dashboard/DashboardContent.tsx

```diff
@@ -25,7 +25,7 @@ export default function DashboardContent() {
       <div className="flex w-full justify-between gap-2">
         <DropdownMenu>
           <DropdownMenuTrigger asChild>
-            <Button className="rounded-xl px-4 py-3 text-lg font-bold text-white hover:bg-primary-800">
+            <Button className="rounded-xl bg-gradient-linear from-sky-300 to-primary-700 px-4 py-3 text-lg font-bold text-white shadow-none hover:bg-primary-800">
```

Comment:
can you make a custom color with gradient in the tailwind.config 
Declare a custom linear gradient color and make that reusable